setuptools_rust/setuptools_ext.py

```diff
@@ -17,6 +20,56 @@
 
 
 def add_rust_extension(dist):
+    sdist_base_class = dist.cmdclass.get("sdist", sdist)
+    sdist_options = sdist_base_class.user_options.copy()
+    sdist_boolean_options = sdist_base_class.boolean_options.copy()
+    sdist_negative_opt = sdist_base_class.negative_opt.copy()
+    sdist_options.extend([
+        ('vendor-crates', None,
+         "vendor Rust crates"),
+        ('no-vendor-crates', None,
+         "don't vendor Rust crates."
+         "[default; enable with --vendor-crates]"),
+    ])
+    sdist_boolean_options.append('vendor-crates')
+    sdist_negative_opt['no-vendor-crates'] = 'vendor-crates'
+
+    class sdist_rust_extension(sdist_base_class):
+        user_options = sdist_options
+        boolean_options = sdist_boolean_options
+        negative_opt = sdist_negative_opt
+
+        def initialize_options(self):
+            super().initialize_options()
+            self.vendor_crates = 0
+
+        def get_file_list(self):
+            if self.vendor_crates:
+                manifest_paths = []
+                for ext in self.distribution.rust_extensions:
+                    manifest_paths.append(ext.path)
+                if manifest_paths:
+                    base_dir = self.distribution.get_fullname()
+                    dot_cargo_path = os.path.join(base_dir, ".cargo")
+                    self.mkpath(dot_cargo_path)
+                    cargo_config_path = os.path.join(dot_cargo_path, "config.toml")
+                    vendor_path = os.path.join(dot_cargo_path, "vendor")
+                    command = [
+                        "cargo", "vendor", "--manifest-path", manifest_paths.pop(0)
+                    ]
+                    if manifest_paths:
+                        command.append("--sync")
+                        command.extend(manifest_paths)
+                    command.append(vendor_path)
```

Comment:
Does ``--sync`` accept multiple paths or does ``--sync`` require multiple repetitions? I would expect that you have to do ``--manifest-path path1/Cargo.toml --sync path2/Cargo.toml --sync path3/Cargo.toml``.

```suggestion
                    command = [
                        "cargo", "vendor", "--manifest-path", manifest_paths[0]
                    ]
                    # additional Cargo.toml for extension 1..n
                    for extra_path in manifest_paths[1:]:
                        command.append("--sync")
                        command.append(extra_path)
                    command.append(vendor_path)
```

Reply:
Seems that both should work, https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.multiple